tendrl-cephd.service

```diff
@@ -1,14 +0,0 @@
-[Unit]
```

Comment:
This file is moved to systemd directory. I forget to add it here. I will add in the next patch.